crates/sui-indexer-alt/src/pipeline/sequential/mod.rs

```diff
@@ -41,6 +42,13 @@ pub trait Handler: Processor {
     /// If at least this many rows are pending, the committer will commit them eagerly.
     const MIN_EAGER_ROWS: usize = 50;
 
+    /// If there are more than this many rows pending, the committer will only commit this many in
+    /// one operation. The size is chosen to maximize the rows without hitting the limit on bind parameters.
+    const MAX_INSERT_CHUNK_ROWS: usize = i16::MAX as usize / Self::Value::FIELD_COUNT;
+
+    /// Each deletion will include at most this many rows without hitting the limit on bind parameters.
+    const MAX_DELETE_CHUNK_ROWS: usize = i16::MAX as usize;
+
```

Comment:
What I had meant in my earlier comment was that we shouldn't try to generalize the chunk limits across sequential pipelines, because they are responsible for too much of their commit implementation. Instead we should leave the constants in each implementation, but just implement them in terms of `FIELD_COUNT`.